### DISCUSSION TITLE
Install airflow on windows server 2016

### DISCUSSION THREAD

**Discussion by kumarabhisek91** [apache/airflow#31199] [General]
Can anyone suggest how to install airflow on windows server 2016 with docker

**Comment by potiuk** [comment]
Airflow does not support running even in docker on Windows so if you have any problems, nobody will be able to help you.

Your best bet currently with installing Airlfow is to use WSL2 (with a number of limitations as well) but I think even running WSL2 on 2016 server will have many problems.

My recommendation: DON'T DO THAT for anything even closely related to production. It will break and stop working in completely unexpected ways that nobody will be able to help you with.

You've been warned.
